src/Countries/Sweden.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Sweden extends Country
+{
+    public function countryCode(): string
+    {
+        return 'se';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Nyårsdagen' => '01-01',
+            'Trettondedag jul' => '01-06',
+            'Första maj' => '05-1',
+            'Nationaldagen' => '06-6',
+            'Juldagen' => '12-25',
+            'Annandag jul' => '12-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = $this->easter($year)->setTimezone('Europe/Stockholm');
+
+        /** @var CarbonImmutable $midsummerDay */
+        $midsummerDay = CarbonImmutable::createFromFormat('Y-m-d', "{$year}-06-20", 'Europe/Stockholm');
```

Comment:
Timezones shouldn't be needed. You can call ->startOfDay() on the Carbon instance.

Example: https://github.com/spatie/holidays/blob/main/src/Countries/Country.php#L52

Reply:
I saw that some countries added timezones, and some do not. 

https://github.com/spatie/holidays/blob/914ae68e4dd979172182fb60058ae914db12c0f8/src/Countries/Australia.php#L33

Reply:
Thanks, I fixed some of them in the meantime.